### DISCUSSION TITLE
Save python pickle object to duck db an example please.

### DISCUSSION THREAD

**Discussion by saisaigraph** [duckdb/duckdb#9439] [Q&A]
Could anyone help me with Saving python pickle object to duckdb example please.
Thanks
Sai

**Comment by Mause** [comment]
What issues are you having?

**Comment by yoonghm** [comment]
I suggest to store links (file or directory paths) to python pickle objects. With that you can update and Python could read natively. Putting too many blob objects may affect performance.
